### PR TITLE
Merchantstuff2

### PR DESCRIPTION
**Pull request by Spummm**
Git user: Spummm
Discord/ckey: Spum
Changes:
Moved merchant station powerloader so it will stop dying to simplemobs
Energy weapons are now worth MUCH more
Removed useless wizard trader that only buys wizard items
Modified animal trader so buying certain animals actually works now
Added several additional items that can be sold by diona and history merchant
Added the ability to buy random seeds from ONE trader that doesn't always show up
Modified other two food traders, one sells (most) drinks, the other sells (most) food items
Added some additional items to AI merchants

### CODE REVIEW

**Comment by Spummm** [comment]
This still compiles, runs, and works as intended.

File: maps/torch/z1_admin.dmm [hunk 1]
```diff
@@ -11061,6 +11054,10 @@
 /obj/machinery/atmospherics/pipe/simple/visible{
 	dir = 4
 	},
+/mob/living/exosuit/premade/powerloader{
+	health = 1000;
```
Comment (Kaksisilma) [inline maps/torch/z1_admin.dmm]:
```suggestion

```

File: maps/torch/z1_admin.dmm [hunk 2]
```diff
@@ -11061,6 +11054,10 @@
...
+	maxHealth = 1000
```
Comment (Kaksisilma) [inline maps/torch/z1_admin.dmm]:
```suggestion

```

**Review by Kaksisilma — CHANGES_REQUESTED** [review]
Still gonna ask for @Yawet330 to take a look at the worths_list changes since I'm not sure what impact they'll have.

**Pull request merged**